karma.conf.js

```diff
@@ -18,7 +18,6 @@ module.exports = function (config) {
 
     // list of files / patterns to exclude
     exclude: [
-      './tests/api/state/stateManager.js' // 4, "# should clear the cache when the state root is set"
```

Comment:
Why is this removed?

Reply:
This was just a test, since the reason for your testing PR is to get these browser tests to pass (at least asyncawait dependency was one is eventual several blockers). Coverage decrease is also happening without this commit.